scripts/inject/index.ts

```diff
@@ -71,7 +71,7 @@ let platform: DiscordPlatform | undefined;
             platform = current;
             break;
           }
-        } catch (e) {}
+        } catch (_e) {}
```

Comment:
Let's just remove the catch argument for all of these cases (not gonna comment on each one).
```suggestion
        } catch {}
```